### ISSUE TITLE
Some elements are not announced correctly by TalkBack

### ISSUE THREAD

**Issue by Versty** [AdguardTeam/AdguardForAndroid#4809]
### Version
AdGuard For Android 4.0 

### Environment
Any compatible android version 

### Steps to reproduce 
1. Enable TalkBack in the system settings 
2. Launch AdGuard 
3. Click the ad blocking icon on the hame tab 

### Expected behaviour 
The element is voiced as an "ad blocking protection module checkbox; checked" 

### Actual behaviour
The element is voiced as a "checked; checkbox", the actual puprose of the element is not announced. 

### Additional info 
The same is applicable to other icons on the Home screen, to the protection and statistics tab icons;
Radio buttons is announced as an "unlabelled" in the whole app.
